### ISSUE TITLE
Add custom debug information from anywhere in the application/modules

### ISSUE THREAD

**Issue by L1lle** [biakaveron/debug-toolbar#2]
Is there a way to add in a custom tab information from anywhere in the application?

Like showing debug information of own modules or classes in an extra tab.

For example in a static function:

```
Debugtoolbar::add_tab('tabname',Kohana::debug($some_variable));
```

I don't want to change your module if there is anything like this planed.

**Comment by biakaveron** [comment]
Implemented in fc24a6154bbf86b03fdf. Use DebugToolbar::add_custom('tabname', $var) and if it is an array or object, Kohana::dump() will be applied.

**Comment by L1lle** [comment]
wow, that was quick. thanks a lot. i will check it out tomorrow.
